src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -58,6 +60,7 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 1.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@original-brownbear I never did a puzzle so far, could you please tell me what I'm supposed to do exactly?